keras/applications/inception_resnet_v2.py

```diff
@@ -177,6 +177,7 @@ def InceptionResNetV2(include_top=True,
                       input_tensor=None,
                       input_shape=None,
                       pooling=None,
+                      pretrained_weights_path=None,
```

Comment:
I would instead suggest overloading the keyword `weights`.

If `weights` matches a preset string (imagenet): load that
Else: check if `os.path.exists(weights)`. If does, load it. Else, raise appropriate error, saying "weights did not match "imagenet", not was it a valid path a file on disk"

Reply:
Thanks.  I updated the PR to take the path as the `weights` argument.  I still allow `None` as a valid argument, which I assume is what you intended in your comment.

I manually re-ran my local test on the new commit (verifying same prediction results from each model when called with `weights='imagenet'` and then with `weights` set to another path with a copy of the weights file.)